### PR TITLE
feat: improve test and describe rendered title #70

### PR DESCRIPTION
**Pull request by amiceli**

### CODE REVIEW

File: src/vitest/describe/scenarioOutline.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export function describeScenarioOutline (
                 .checkIfStepExists(stepType, stepDetails)
 
             scenarioStepsToRun.push({
-                key : `${stepType} ${stepDetails}`,
+                key : `${stepType}: ${stepDetails}`,
```
Comment (charlieforward9) [inline src/vitest/describe/scenarioOutline.ts:45]: Following the standard Cucumber syntax, it makes sense to leave the colon out of the step key.

> ["When Cucumber tries to execute a step, it looks for a matching step definition to execute."](https://cucumber.io/docs/gherkin/reference/#:~:text=when%20cucumber%20tries%20to%20execute%20a%20step%2C%20it%20looks%20for%20a%20matching%20step%20definition%20to%20execute.)

```suggestion
                key : `${stepType} ${stepDetails}`,
```

**Review by charlieforward9 — CHANGES_REQUESTED** [review]

File: src/vitest/describe/scenarioOutline.ts (hunk 1 above)
Comment (amiceli) [reply src/vitest/describe/scenarioOutline.ts:45]: Yes, I wanted to add colon to Scenario etc.
I will remove it for steps ;).

**Comment by amiceli** [comment]
I try with vitest vscode extension, it's ok.
I'm going to publish a new version.

**Comment by charlieforward9** [comment]
Before you finish, is there a simple way of preserving the step order in the test navigator? My steps are rendering out of order.

**Comment by amiceli** [comment]
I publish `3.2.5`, I saw your message too late ;).

For steps order, I've same thing on vscode, for example `Then` before `When`.
I thing it's an alphabetically order in vitest extension.

**Pull request merged**